README.md

```diff
@@ -10,10 +10,43 @@ The Vault Operator deploys and initializes Vault on Kubernetes, and runs in high
 
 Deploy the charm:
 ```bash
-juju deploy vault-k8s -n 5 --trust
+juju deploy vault-k8s --channel edge -n 5 --trust
 ```
 > Note: It is advised to deploy Vault with an odd number of units
 
+We recommend deploying Vault with an odd number of units.
```

Comment:
We don't need this in addition to the Note just above.

Reply:
This is added mistakenly while resolving conflict.

Reply:
Please don't resolve the conversations when the changes are not pushed. This applies to all of my conversations here.